src/whylogs/core/metrics/nlp_metrics.py

```diff
@@ -0,0 +1,80 @@
+from typing import List, Union
+
+import jiwer
```

Comment:
Can we do `try... import... except` here? I don't want whylogs to have a hard dependency o `jiwer`

Reply:
it is a light dependency, we can seperate when we work with that specific issue.